### PR TITLE
testing/drivertest: add drivertest for cmocka driver testcases

### PR DESCRIPTION
**Pull request by yintao707**
## Summary
Add drivertest for cmocka driver testcases
## Impact
None
## Testing
Pass vela CI

### CODE REVIEW

**Comment by xiaoxiang781216** [comment]
@yintao707 please remove the Gerrit Change-ID's

File: testing/drivertest/drivertest_simple.c [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * Private Functions
+ ****************************************************************************/
+
+static void test_case_01(void **state)
+{
+  (void) state;
+  assert_int_equal(0, 0);
+}
+
+static void test_case_02(void **state)
```
Comment (pkarashchenko) [inline testing/drivertest/drivertest_simple.c:40]:
```suggestion
static void test_case_02(FAR void **state)
```

File: testing/drivertest/drivertest_simple.c [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+#include <stddef.h>
+#include <setjmp.h>
+#include <stdint.h>
+#include <cmocka.h>
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+static void test_case_01(void **state)
```
Comment (pkarashchenko) [inline testing/drivertest/drivertest_simple.c:34]:
```suggestion
static void test_case_01(FAR void **state)
```

File: testing/drivertest/drivertest_simple.c [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+{
+  (void) state;
```
Comment (pkarashchenko) [inline testing/drivertest/drivertest_simple.c:36]:
```suggestion
  UNUSED(state);
```

File: testing/drivertest/drivertest_simple.c [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  assert_int_equal(0, 0);
+}
+
+static void test_case_02(void **state)
+{
+  (void)state;
```
Comment (pkarashchenko) [inline testing/drivertest/drivertest_simple.c:42]:
```suggestion
  UNUSED(state);
```

File: testing/drivertest/drivertest_simple.c (hunk 1 above)
Comment (yintao707) [reply testing/drivertest/drivertest_simple.c]: Done,thankyou

File: testing/drivertest/drivertest_simple.c (hunk 2 above)
Comment (yintao707) [reply testing/drivertest/drivertest_simple.c]: Done,thankyou

File: testing/drivertest/drivertest_simple.c (hunk 3 above)
Comment (yintao707) [reply testing/drivertest/drivertest_simple.c]: Done,thankyou

File: testing/drivertest/drivertest_simple.c (hunk 4 above)
Comment (yintao707) [reply testing/drivertest/drivertest_simple.c]: Done,thankyou

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by xiaoxiang781216** [comment]
let's ignore the Mix Case error

**Pull request merged**